cmd/notation/plugin/install.go

```diff
@@ -330,6 +331,16 @@ func installPluginWithOptions(ctx context.Context, opts plugin.CLIInstallOptions
 		if errors.As(err, &errPluginDowngrade) {
 			return fmt.Errorf("%w.\nIt is not recommended to install an older version. To force the installation, use the \"--force\" option", errPluginDowngrade)
 		}
+
+		var errExeFile *plugin.PluginExecutableFileError
+		if errors.As(err, &errExeFile) {
+			return fmt.Errorf("%w.\nPlease ensure that the plugin executable is compatible with the %s/%s architecture and has the appropriate file permissions.", err, runtime.GOOS, runtime.GOARCH)
```

Comment:
I think it's a bit verbose? How about
```suggestion
			return fmt.Errorf("%w.\nPlease ensure that the plugin executable file is compatible with %s/%s and has appropriate permissions.", err, runtime.GOOS, runtime.GOARCH)
```